### ISSUE TITLE
Cannot search encrypted data

### ISSUE THREAD

**Issue by alisamdaanali** [gregoryduckworth/Encryptable#3]
I am unable to search the data using the ->where()

**Comment by gregoryduckworth** [comment]
Have you added the trait to your model as shown in the Readme?

**Comment by adeguntoro** [comment]
Yeah i did everything like on Readme. The data is encrypted and when i use my view i see my data like "normal" data. I use "where" condition, but the result is empty.
```
$data = MyModel::where('nik', '1234')->get();
return response()->json($data);
```

**Comment by bumbummen99** [comment]
> I am unable to search the data using the ->where()

That is since it is encrypted with the app key, you can not search encrypted data and you can not use this for searchabale data. And this holds true for all packages that do this.
